internal/pkg/cli/pipeline_update.go

```diff
@@ -103,6 +103,75 @@ func (o *updatePipelineOpts) Validate() error {
 	return nil
 }
 
+// Execute creates a new pipeline or updates the current pipeline if it already exists.
+func (o *updatePipelineOpts) Execute() error {
```

Comment:
Note: there are no changes to `Execute()`-- in the first commit I just moved this method higher up in the file.